src/components/PipelineRunDetailsView/visualization/utils/pipelinerun-graph-utils.ts

```diff
@@ -118,6 +118,31 @@ export const appendStatus = (
     } else if (mTask.status && !mTask.status.reason) {
       mTask.status.reason = RunStatus.Pending;
     }
+
+    // Determine any task test status
+    const taskRunName =
+      pipelineRun.status.taskRuns &&
+      Object.keys(pipelineRun.status.taskRuns).find(
+        (key) => pipelineRun.status.taskRuns[key].pipelineTaskName === mTask.name,
+      );
+    const taskRun = taskRunName && pipelineRun.status.taskRuns[taskRunName];
+    if (taskRun?.status?.taskResults) {
+      const testOutput = taskRun?.status?.taskResults.find(
+        (result) => result.name === 'HACBS_TEST_OUTPUT',
+      );
+      if (testOutput) {
+        try {
+          const outputValues = JSON.parse(testOutput.value);
+          if (outputValues.result === 'FAILURE') {
+            mTask.testStatus = RunStatus.Failed;
```

Comment:
After seeing this in action, showing failed nodes in visualization and success (actual Tekton status) in the logs tab could create confusions (Tekton will stop execution of following tasks if there is a real failure but in this case it may look like it continued execution).  I think Its best we just show a warning to the user if there is a CVE failure/warning and then redirection to logs should help the user to see more details. wdyt?

<img width="367" alt="Screenshot 2023-02-27 at 10 39 20 AM" src="https://user-images.githubusercontent.com/9964343/221479365-3b3c9808-5117-461a-849e-5617a40223f2.png">
<img width="1414" alt="Screenshot 2023-02-27 at 10 39 41 AM" src="https://user-images.githubusercontent.com/9964343/221479372-ec2ea1a1-8bcd-4967-bd80-d4057f3a9262.png">

Reply:
I would agree